packages/react/src/components/avatar/avatar.tsx

```diff
@@ -191,7 +173,11 @@ const AvatarImage: FC<AvatarImageProps> = ({
       loading={loading}
       referrerPolicy={referrerPolicy}
       rounded={rounded}
-      __css={css}
+      __css={{
+        height: "100%",
+        objectFit: "cover",
+        width: "100%",
+      }}
     />
```

Comment:
@chida09 
Is it necessary to define the object in `packages/react/src/theme/components/avatar.ts` as well as others?
Is it ok to still have CSS defined in component?

Reply:
@hirotomoyamada @odendayoko
When migrating styles to theme, how should we handle cases where the styles variable isn't being used? I see two potential options:

1. Create new properties in the theme like styles.image (though this would currently cause type errors)
2. Leave these styles in components without migrating them to theme in this PR

Which approach would you recommend for this migration?

Reply:
@chida09 @odendayoko 

No need to create a new key. If you don't want to port it to a theme and just remove the `css` variable and move it into the file, it is not necessary.